test/common/tcp_proxy/upstream_test.cc

```diff
@@ -207,14 +210,23 @@ template <typename T> class HttpUpstreamRequestEncoderTest : public testing::Tes
   }
 
   void setupUpstream() {
-    config_ = std::make_unique<TunnelingConfigHelperImpl>(config_message_);
+    config_ = std::make_unique<TunnelingConfigHelperImpl>(config_message_, context_);
     upstream_ = std::make_unique<T>(callbacks_, *this->config_, this->downstream_stream_info_);
   }
 
+  void populateMetadata(envoy::config::core::v3::Metadata& metadata, const std::string ns,
+                        const std::string key, const std::string value) {
```

Comment:
nit: these could be `const std::string&`

Reply:
Done